<?xml version="1.0" encoding="UTF-8"?>
<testsuites>
  <testsuite name="exampletests" package="exampletests" errors="0" tests="6" failures="3" timestamp="2023-04-11T12:28:23" time="0">
    <testcase classname="my machine" name="Checking user &quot;www-data&quot; exists" time="0">
    </testcase>
    <testcase classname="my machine" name="Checking user &quot;mysql&quot; exists" time="0">
      <failure type="unit-test" message="failed"><![CDATA[Checking user "mysql" exists: FAIL]]></failure>
    </testcase>
    <testcase classname="my machine" name="Checking process &quot;udchcpd&quot; running as any user" time="0">
      <failure type="unit-test" message="failed"><![CDATA[Checking process "udchcpd" running as any user: FAIL]]></failure>
    </testcase>
    <testcase classname="my machine" name="Checking process &quot;Xorg&quot; running as any user" time="0">
    </testcase>
    <testcase classname="my machine" name="Checking process &quot;nginx&quot; running as www-data" time="0">
      <failure type="unit-test" message="failed"><![CDATA[Checking process "nginx" running as www-data: FAIL]]></failure>
    </testcase>
    <testcase classname="my machine" name="Checking python package &quot;math&quot; is available" time="0">
    </testcase>
  </testsuite>
</testsuites>
